ui/app/AppLayouts/Chat/stores/RootStore.qml

```diff
@@ -60,6 +61,10 @@ QtObject {
             expression: icon(model.icon)
         }
     }
+
+    readonly property bool isUserAllowedToSendMessage: _d.isUserAllowedToSendMessage
+    readonly property string chatInputPlaceHolderText: _d.chatInputPlaceHolderText
+    readonly property var oneToOneChatContact: _d.oneToOneChatContact
```

Comment:
Ahh, actually alias is not working here. I think is because of the inlined extra properties:
https://doc.qt.io/qt-5/qtqml-syntax-objectattributes.html#property-aliases-and-types